Workbooks/Images/Logos/illusive_logo_workbook.svg

```diff
@@ -1 +1,2 @@
\ No newline at end of file
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 126 179.97"><defs><style>.cls-1{fill:#ff9e29;fill-rule:evenodd;}</style></defs><g id="illusive_symbol" data-name="illusive symbol"><path id="Path" class="cls-1" d="M123.09,168.15A10,10,0,0,1,119,170.3a23.67,23.67,0,0,1-6.22.71H72.28a7.25,7.25,0,0,0-3.42,1.05,4.18,4.18,0,0,0-1.49,2,8.59,8.59,0,0,0-.44,2.71,22.91,22.91,0,0,1-.28,3.2H59.42a23.27,23.27,0,0,1-.26-2.5,17.65,17.65,0,0,0-.19-2.18,6.32,6.32,0,0,0-.53-1.8,3.17,3.17,0,0,0-1.26-1.33,7.5,7.5,0,0,0-2.39-.85,20.9,20.9,0,0,0-3.87-.3H13.2A23.57,23.57,0,0,1,7,170.3a10,10,0,0,1-4.09-2.15,8.25,8.25,0,0,1-2.23-3.54A16.2,16.2,0,0,1,0,159.67c0-1.18,0-2.22.05-3.12s.09-1.67.14-2.3.1-1.37.15-1.93H12a13.64,13.64,0,0,0,.36,2.85,3.5,3.5,0,0,0,.93,1.68,3.31,3.31,0,0,0,1.73.79,17.12,17.12,0,0,0,2.77.19h38a14.49,14.49,0,0,1,2.28.17,6.11,6.11,0,0,1,1.91.62,5.39,5.39,0,0,1,1.61,1.28,8.53,8.53,0,0,1,1.34,2.15,7.06,7.06,0,0,1,1.38-2.19,6.16,6.16,0,0,1,1.8-1.28,7.65,7.65,0,0,1,2.1-.6,17.5,17.5,0,0,1,2.29-.15h36.58a21.69,21.69,0,0,0,3.12-.19,3.47,3.47,0,0,0,1.83-.79,3.31,3.31,0,0,0,.91-1.68,14.41,14.41,0,0,0,.34-2.85h12.37c.07.56.14,1.2.19,1.93s.09,1.4.11,2.3,0,1.94,0,3.12a16.2,16.2,0,0,1-.68,4.94,8.25,8.25,0,0,1-2.23,3.54Z" transform="translate(0 0)"/><polygon class="cls-1" points="81.92 90.69 126 90.69 126 104.88 81.92 104.88 81.92 90.69"/><polygon class="cls-1" points="102.34 18.42 94.29 45.21 78.92 45.21 95.38 0 108.77 0 126 45.21 110.99 45.21 102.34 18.42"/><polygon class="cls-1" points="47.61 114.11 35.42 122.96 26.54 110.64 17.71 122.96 5.59 114.11 14.47 101.85 0 97.25 4.69 83.02 19.09 87.68 19.09 72.61 34.11 72.61 34.11 87.68 48.46 83.02 53.14 97.25 38.67 101.85 47.61 114.11 47.61 114.11"/><polygon class="cls-1" points="23.42 18.42 15.37 45.21 0 45.21 16.45 0 29.84 0 47.07 45.21 32.07 45.21 23.42 18.42 23.42 18.42"/></g></svg>
```

Comment:
Please copy the contents of Logos/illusiveAttackManagementSystem.svg into this logo file as well. 
If you want to keep the logos different, then please do the following changes to this logo.
1. Format the logo file
2. Remove style and class tags as these cause issues in deploying to Azure

Reply:
also please remove `data-name` attributes

Reply:
We copied the content of Logos/illusiveAttackManagementSystem.svg